asv/template/asv.conf.json

```diff
@@ -13,6 +13,9 @@
     // project being benchmarked
     "repo": "",
 
+    // List of branches to benchmark. If not provided, defaults to "master".
+    // "branches": ["master"],
```

Comment:
It probably should be `default` for mercurial -- `tip` is something more similar to `HEAD` as it always points to the most recent commit.